lib/sshutils/sftp/local.go

```diff
@@ -36,91 +37,112 @@ func (l localFS) Type() string {
 	return "local"
 }
 
-func (l localFS) Stat(ctx context.Context, path string) (os.FileInfo, error) {
+func (l *localFS) Glob(ctx context.Context, pattern string) ([]string, error) {
```

Comment:
Nit: Can you add godocs on all exported names?

Reply:
I can, but I thought it was unnecessary due to the fact that `localFS`'s methods are never called directly, only as a `FileSystem` interface which has all of its methods documented. For future reference, in this case is it still necessary to add comments to every method of an implementation of an interface that is (should) never be used directly?